frontend/src/components/WithdrawForm.vue

```diff
@@ -1,55 +1,57 @@
+          appendButtonLabel="Withdraw"
+          :appendButtonDisable="isWithdrawInProgress || isFeeLoading"
+          :appendButtonLoading="isWithdrawInProgress"
+          :disable="isWithdrawInProgress"
+          label="Address"
+          lazy-rules
+          :rules="(val) => (val && val.length > 4) || 'Please enter valid address'"
+        />
+        <!-- Fee estimate -->
+        <div class="q-mb-lg">
+          <div v-if="!isNativeToken && isFeeLoading" class="text-caption text-italic">
+            <q-spinner-puff class="q-my-none q-mr-sm" color="primary" size="2rem" />
+            Fetching fee estimate...
+          </div>
+          <div v-else-if="isNativeToken" class="text-caption">
+            Withdrawal fee: <span class="text-bold"> 0 {{ nativeTokenSymbol }} </span>
+          </div>
+          <div v-else-if="activeFee" class="text-caption">
+            Estimated withdrawal fee:
+            <span class="text-bold">
+              {{ humanizeTokenAmount(activeFee.fee, activeFee.token) }}
+              {{ activeFee.token.symbol }}
+            </span>
+          </div>
         </div>
       </div>
-    </div>
-    <div v-else class="text-center q-mb-lg">
-      <q-spinner-puff class="q-mb-md q-mr-sm" color="primary" size="2rem" />
-      <div class="text-center text-italic">Withdraw in progress...</div>
-    </div>
-
-    <!-- Privacy warning -->
-    <div class="border q-mb-lg" />
-    <div class="text-caption">
-      <q-icon name="fas fa-exclamation-triangle" color="warning" left />
-      <span class="text-bold">WARNING</span>: Be sure you understand the security implications before entering a
-      withdrawal address. If you withdraw to an address publicly associated with you, privacy for this transaction will
-      be lost. <router-link class="hyperlink" to="/faq#receiving-funds" target="_blank"> Learn more </router-link>.
+      <div v-else class="text-center q-mb-lg">
+        <q-spinner-puff class="q-mb-md q-mr-sm" color="primary" size="2rem" />
+        <div class="text-center text-italic">Withdraw in progress...</div>
+      </div>
+      <!-- Privacy warning -->
+      <div class="border q-mb-lg" />
+      <div class="text-caption">
+        <q-icon name="fas fa-exclamation-triangle" color="warning" left />
+        <span class="text-bold">WARNING</span>: Be sure you understand the security implications before entering a
+        withdrawal address. If you withdraw to an address publicly associated with you, privacy for this transaction
+        will be lost.
+        <router-link class="hyperlink" to="/faq#receiving-funds" target="_blank"> Learn more </router-link>.
+      </div>
     </div>
 
     <!-- Advanced feature: show private key -->
     <div v-if="advancedMode">
-      <div @click="emit('togglePrivateKey')" class="text-caption hyperlink q-mt-lg">
+      <div @click="emit('togglePrivateKey')" class="text-caption hyperlink">
         {{ spendingPrivateKey ? 'Hide' : 'Show' }} withdrawal private key
```

Comment:
Wondering if we should change the copy here to "Show/Hide stealth address private key" instead of "withdrawal"

Reply:
Yea I like that phrasing, updated in 5b0a150